cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -19,10 +19,14 @@ import (
 	"github.com/symflower/eval-dev-quality/provider"
 	_ "github.com/symflower/eval-dev-quality/provider/openrouter"
 	_ "github.com/symflower/eval-dev-quality/provider/symflower"
+	"github.com/symflower/eval-dev-quality/tools"
 )
 
 // Evaluate holds the "evaluation" command.
 type Evaluate struct {
+	// InstallToolsPath determines where tools for the evaluation are installed.
+	InstallToolsPath string `long:"install-tools-path" description:"Install tools for the evaluation into this path."`
```

Comment:
Would be cool to have the default location here for the user as well.

(also for the separate command)

Reply:
You mean as the "default value"? We cannot change the description dynamically. Code attributes do not allow that.

Reply:
Yeah I know :crying_cat_face: